src/js/components/stock-movement-wizard/outbound/PickPage.jsx

```diff
@@ -525,6 +535,60 @@ class PickPage extends Component {
       .catch(() => { this.props.hideSpinner(); });
   }
 
+  // Calculating which rows have quantity picked 0, and don't have subitems
```

Comment:
Please either add in the comment that the returned value is a list of indexes, not rows or change the name to indicate it returns row indexes.